lib/ansible/modules/cloud/amazon/ec2_ami_copy.py

```diff
@@ -29,25 +29,25 @@
 options:
   source_region:
     description:
-      - The source region the AMI should be copied from.
+      - the source region that AMI should be copied from
```

Comment:
Ah... Sorry about that, I had originally forked from [ansible-modules-extras](https://github.com/ansible/ansible-modules-extras). Appears someone has since updated the documentation since and I didn't notice.

You can see here: https://github.com/ansible/ansible-modules-extras/blob/devel/cloud/amazon/ec2_ami_copy.py#L32

I'll change them all accordingly.